runtime/onert/api/src/nnfw_api_internal.h

```diff
@@ -149,6 +149,10 @@ struct nnfw_session
   NNFW_STATUS register_custom_operation(const std::string &id, nnfw_custom_eval eval_func);
   NNFW_STATUS input_tensorindex(const char *tensorname, uint32_t *index);
   NNFW_STATUS output_tensorindex(const char *tensorname, uint32_t *index);
+  /**
+   * @brief   Set backends from string-encoded mappings from operation index to backend type 
```

Comment:
```suggestion
   * @brief   Set backends from string-encoded mappings from operation index to backend type
```

Reply:
@hseok-oh format-check failed after committing your suggestion. Let me revise the description. I want to repeated `from`.

Reply:
Ah, it is your 2nd suggestion to fix. The problem was the trailing space. I see.